docs/crio.8.md

```diff
@@ -276,15 +276,15 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--seccomp-use-default-when-empty**="": Use the default seccomp profile when an empty one is specified (default: false)
 
-**--selinux**: Enable selinux support (default: false)
+**--selinux**: Enable selinux support (default: true)
```

Comment:
unfortunately, our docs generation messes with these values. It seems you can just drop the changes to this file from the commit.

It may be easier to just run 
`sed -i 's|/var/run/|/run|g' docs/crio.8.md`

Reply:
Only apply this /var/run to /run change for /var/run/containers/storage, or all the /var/run/crio/* in this readme should be change to /run/*?  It looks like the failure only complains about the **--runroot**

Reply:
sorry, you're right, we only need it for the run root